src/components/App.js

```diff
@@ -1,11 +1,26 @@
 import React, { Component } from 'react';
+import {BrowserRouter,Route,Switch} from 'react-router-dom';
+import {Header} from 'react-fullpage';
+
 import '../styles/App.css';
+import HomePage from './homepage/HomePage';
 import Comming from './comming-soon/CommingSoon';
+import Navbar from './Navbar';
 
 class App extends Component {
 	render() {
 		return (
-			<Comming/>
+			<BrowserRouter>				
+				<div className='main-div'>
+					<Header>
+						<Navbar/>
+					</Header>
+					<Switch>
+						<Route path='/' component={HomePage} exact/>
+						<Route component={Comming}/>
```

Comment:
Don't redirect to `ComingSoon` page, instead put `Coming Soon` text when some click on `Events`, `Talks`, `Teams` and other tabs. Use the same background color which is used in `HomePage`